applications/FluidDynamicsApplication/custom_utilities/fluid_adjoint_variable_information.h

```diff
@@ -0,0 +1,209 @@
+//    |  /           |
+//    ' /   __| _` | __|  _ \   __|
+//    . \  |   (   | |   (   |\__ `
+//   _|\_\_|  \__,_|\__|\___/ ____/
+//                   Multi-Physics
+//
+//  License:         BSD License
+//                   Kratos default license: kratos/license.txt
+//
+//  Main authors:    Suneth Warnakulasuriya
+//
+
+#if !defined(KRATOS_FLUID_ADJOINT_VARIABLE_INFORMATION_H)
+#define KRATOS_FLUID_ADJOINT_VARIABLE_INFORMATION_H
+
+// System includes
+#include <array>
+
+// External includes
+
+// Project includes
+#include "includes/variables.h"
+
+// Application includes
+#include "custom_utilities/fluid_adjoint_utilities.h"
+#include "fluid_dynamics_application_variables.h"
+
+namespace Kratos
+{
+///@addtogroup FluidDynamicsApplication
+///@{
+
+namespace AdjointVariableInformation
+{
```

Comment:
Remember the `namespace`s discussion. Creation of new `namespaces` would only be allowed in very specific cases under the agreement of the @KratosMultiphysics/technical-committee. Do you really need this one? If so it is, can you explain a bit?

Reply:
I remember and I understand. In here I'm using the namespace just to encapsulate few classes [ To make things little bit tidy, and this same namespace will have some more additional classes for RANS in next PRs. ]. So I'm not having problems with `KRATOS_API` nor method exposure to the namespace. Would this be ok?

Reply:
@KratosMultiphysics/technical-committee

Reply:
Removed this namespace and the classs 👍